### DISCUSSION TITLE
FocusScope blocks stripe and feedback modals

### DISCUSSION THREAD

**Discussion by neguezaymen** [adobe/react-spectrum#4364] [General]
Hi,
I use FocusScope for my different modals and it works very well, the problem is that I also use usabilla modal for feedback and stripe for payment and when I have a modal open with FocusScope and I launch the usabilla modal or Stripe the focus remains blocked on the other modal and I can't even write in an input and I can't do anything.
Please help

**Comment by snowystinger** [comment]
I haven't used Stripe, but couple questions for you. Do you control the Stripe's modal? As in do you have a reference to the HTML element that is the Stripe modal. Or do you control what HTML element the Stripe modal renders into? Discussing how we potentially want to handle this, but we don't know the use cases well enough yet.

**Comment by neguezaymen** [comment]
Hi,
I'm talking about the mode for payment verification by sms managed by stripe and for usabilla it's a script injected by
dangerouslySetInnerHTML.
in these 2 cases when I have a modal with an open Focus Scope the focus remains blocked inside

**Reply by snowystinger** [reply]
Do you know if there is a demo place where I can try each of these two cases? Or the specific documentation for them? I'm trying to gather the possible control mechanisms for these payment modals so we know what options we have.
